pom.xml

```diff
@@ -9,14 +9,22 @@
     <version>1.0-SNAPSHOT</version>
 
     <properties>
-        <java.version>1.8</java.version>
-        <selenium.version>3.14.0</selenium.version>
-        <cdt-java-client.version>2.1.0</cdt-java-client.version>
-        <junit-jupiter.version>5.6.0</junit-jupiter.version>
+        <java.version>17</java.version>
+        <selenium.version>4.1.4</selenium.version>
+        <cdt-java-client.version>5.0.0</cdt-java-client.version>
+        <junit-jupiter.version>5.8.2</junit-jupiter.version>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <slf4j-log4j12.version>1.7.30</slf4j-log4j12.version>
+        <slf4j-log4j12.version>2.0.0-alpha7</slf4j-log4j12.version>
     </properties>
 
+    <repositories>
```

Comment:
Do we need this? Seems to be useless.

Reply:
Right, I initially tested it from within local repo.

Reply:
Done